lib/kafka_ex/config.ex

```diff
@@ -9,6 +9,8 @@ defmodule KafkaEx.Config do
 
   require Logger
 
+  @default_port 9092
```

Comment:
I don't think we should be setting a default port, we should either log in debug mode if the pattern for `host:port` is not matched, ignore it or error